pkg/peertls/tlsopts/tls.go

```diff
@@ -36,11 +35,18 @@ func (opts *Options) ServerOption() grpc.ServerOption {
 	return grpc.Creds(credentials.NewTLS(tlsConfig))
 }
 
-// DialOption returns a grpc `DialOption` for making outgoing connections
+// DialIDVerifiedOption returns a grpc `DialOption` for making outgoing connections
 // to the node with this peer identity.
-// id is an optional id of the node we are dialing.
-func (opts *Options) DialOption(id storj.NodeID) grpc.DialOption {
-	return grpc.WithTransportCredentials(opts.TransportCredentials(id))
+func (opts *Options) DialIDVerifiedOption(id storj.NodeID) (grpc.DialOption, error) {
```

Comment:
I think these should be reversed. `DialUnverifiedOption` and `DialOption`.

Reply:
Good point